src/components/datePicker/DatesSelectionConnected.tsx

```diff
@@ -29,10 +29,10 @@ const mapStateToProps = (state: IReactVaporState, ownProps: IDatesSelectionOwnPr
   return {
     lowerLimit: item ? item.lowerLimit : new Date(),
     upperLimit: item ? item.upperLimit : new Date(),
-    inputLowerLimit: item && item.inputLowerLimit,
-    inputUpperLimit: item && item.inputUpperLimit,
-    quickOption: optionPicker && optionPicker.selectedValue,
-    isSelecting: item && item.selected,
+    inputLowerLimit: item ? item.inputLowerLimit : new Date(),
+    inputUpperLimit: item ? item.inputUpperLimit : new Date(),
+    quickOption: optionPicker ? optionPicker.selectedValue : '',
```

Comment:
why not undefined anymore?

Reply:
I had an error in the terminal while running UTs. I stumbled accross this error and this is the way to fix it. (see https://github.com/facebook/react/issues/6895)

Reply:
You should add these to the default props instead